frontend/components/round.tsx

```diff
@@ -0,0 +1,23 @@
+"use client";
+
+import { useEffect, useState } from "react"
+
+export default function Round({ gameState }) {
```

Comment:
it doesn't look like the round updates after each round.

I think maybe pass in the socket and call `setRound` everytime a `updateGameState` event happens